packages/expo-file-system/src/next/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FileSystem';
```

Comment:
Let's export only a subset of things that we really want to be exported. Even if we want to export everything right now, I'm in favor of explicitly listing them instead of using `*`, so the public API is clearly defined here.